tailwind.config.js

```diff
@@ -51,6 +53,7 @@ module.exports = {
             steel: "rgb(143, 143, 143)",
             dark: "rgb(40, 40, 40)",
             fairy: "hotpink",
+            "white-op": "rgba(255, 255, 255, 0.2)",
```

Comment:
This is not necessary. Instead you can use the already registered `white` color and [add some opacity](https://tailwindcss.com/docs/text-decoration-color#changing-the-opacity) to it.